01LessonOne/LessonOne.js

```diff
@@ -14,14 +16,27 @@ module.exports = function() {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
 
+   if (big > small){
```

Comment:
What kind of value does the expression `big > small` result in? What are the possible values? What are the circumstances that lead to the expression being a particular value?